push/tests.py

```diff
@@ -4,25 +4,49 @@
 import json
 
 
-class PushNotificationApiTest(APITestCase):
+class SubscriptionApiTest(APITestCase):
+    url = reverse('push:subscribe')
```

Comment:
write this within a `setup` method. https://docs.djangoproject.com/en/3.2/topics/testing/overview/